distribution/docs/pom.xml

```diff
@@ -21,13 +21,16 @@
     </parent>
     <artifactId>docs</artifactId>
     <name>DDF DOCS</name>
+    <properties>
+        <skipTests>true</skipTests>
+    </properties>
     <build>
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
-                    <skip>true</skip>
+                    <skip>${skipTests}</skip>
```

Comment:
the property is so I can flip this depending on whether the profile runs or not. If it's just true, it'll always fail incremental builds.